test/integration/test-video-players-helper.js

```diff
@@ -256,12 +256,16 @@ export function runVideoPlayerIntegrationTests(
   }
 
   function cleanUp() {
```

Comment:
looked into it. It doesn't cleanup iframes in Safari for some reason..
from `_init_tests.js_`
```
  if (!platformFor(window).isSafari()) {
    cleanupTagNames.push('iframe');
  }
```

let me remove this and enable cleanup it for Safari and see how it goes.